bolt_transport.go

```diff
@@ -255,6 +255,11 @@ func (t *BoltTransport) dispatchHistory(s *Subscriber, toSeq uint64) {
 			}
 		}
 		s.HistoryDispatched(responseLastEventID)
+		if !afterFromID {
+			if c := t.logger.Check(zap.ErrorLevel, "Can't find LastEventID in Bolt DB"); c != nil {
+				c.Write(zap.Error(ErrIdNotFound))
```

Comment:
This isn't necessary to create an error type as we don't use it. You can log a string directly.